src/lib/operators/sort.cpp

```diff
@@ -208,7 +208,11 @@ class Sort::SortImpl : public AbstractReadOnlyOperatorImpl {
 
     const auto chunk_count = output->chunk_count();
     for (auto chunk_id = ChunkID{0}; chunk_id < chunk_count; ++chunk_id) {
-      output->get_chunk(chunk_id)->set_ordered_by(std::make_pair(_column_id, _order_by_mode));
+      const auto chunk = output->get_chunk(chunk_id);
+      if (chunk->is_mutable()) {
+        chunk->finalize();
```

Comment:
I checked and the assert fails e.g. during test `OperatorsAggregateTest/1.AnyOnGroupWithMultipleEntries` since the tests never finalize the chunks loaded.
It might be a good idea to make finalization a requirement though instead of handling this in sort, which might be unexpected. We'll discuss this.

Reply:
Never mind, we're creating the chunks here. I guess we can skip the mutable check then.

Reply:
Changed in 76adbb61ea654e755a31eaa790dafb4690e02975 (I can't mark discussions as resolved)

Reply:
Yepp, unfortunately, Github's rights management is very inflexible. We can't allow people to add tags or resolve comments without also allowing them pushes to the master...